day02/ex05/ex05.md

```diff
@@ -0,0 +1,34 @@
+# Exercise 05 - Flask API - Download.
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turn-in directory:    |  ex05              |
+|   Files to turn in:     |  app.py, s3_funcs.py |
+|   Forbidden function:   |  None              |
+|   Remarks:              |  n/a               |
+
+
+To finish our Flask API we will implement one last functionnality, the download. For this exercise, we are going to avoid the proxy solution. Indeed, we have an easier and faster solution to do that. Do you remember ? Yes, I am talking about presigned URLs !
```

Comment:
```suggestion
To finish our Flask API we will implement one last functionality, the download. For this exercise, we are going to avoid the proxy solution. Indeed, we have an easier and faster solution to do that. Do you remember? Yes, I am talking about presigned URLs !
```